packages/react/package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@clerk/shared": "^0.20.0",
     "@clerk/types": "^3.48.1",
-    "swr": "1.3.0",
+    "swr": "2.2.0",
```

Comment:
Let's rebase from main as this has been removed already :)

Reply:
Cool, thanks :)